ush/aqm/py_plotting_ozmax8.config

```diff
@@ -0,0 +1,159 @@
+#                   plot all of the listed models in MODEL on the same plot, as well as all of the 
+#                   init/valid hours in FCST_INIT/VALID_HOUR and lead times in FCST_LEAD.  
+#                   Exceptions to this are the listed var_names and listed domains (VX_MASK_LIST), 
+#                   for which individual plots will be made. Listed levels are also plotted 
+#                   separately unless the plot type is stat_by_level.
+#
+# EXECUTION:        After configuring this script, it can be run on the command line:
+#
+#                   $ /bin/sh ${PY_PLOT_DIR}/py_plotting.config
+#
+#                   ... which will set the environment variables and run the python code. The python 
+#                   code then follows these steps:
+#                   (1) Save environment variables as global python variables. Make sure the 
+#                       datatypes are correct
+#                   (2) Check the user settings and throw errors if an issue is encountered
+#                   (3) Send the settings to df_preprocessing.py, which pulls and prunes the .stat 
+#                       files--creating temporary data files in the process and storing those in 
+#                       PRUNE_DIR--then loads the data as pandas dataframes, which are filtered 
+#                       several times according to user settings. 
+#                   (4) Send the dataframe and user settings to a plotting function, which creates a 
+#                       figure object, filters the data, plots the data, adjusts plot features, then 
+#                       saves the plot as a png.
+#
+# DEBUGGING:        Part of the configuration for this script involves setting a logfile path. 
+#                   Running the script will print the logfile path for your specific task, which you 
+#                   can check for debugging.  The lowest log level that is currently functional is 
+#                   "INFO". "DEBUG" statements may be included in a later implementation.
+#                
+# ================================================================================================
+set -x
+
+# Case type settings are used to check if settings are allowed. **
+export VERIF_CASE="grid2obs"
+export VERIF_TYPE="aq"
+
+# Used to name the graphics ouput files. File names will start with the URL_HEADER, followed by plot 
+# details.  Leave blank ("") to include only plot details in the file names.
+export URL_HEADER=""
+
+# Directory Settings.  Make sure USH_DIR, PRUNE_DIR, and SAVE_DIR point to desired .../ush, 
+# .../data, and .../out directories.  OUTPUT_BASE_DIR is the location of the metplus stat archive.  
+# The format must be ${OUTPUT_BASE_DIR}/${VERIF_CASE}/${MODEL_NAME}/${Ym}/${MODEL_NAME}*.stat
+# The format requirement can be changed by editing the ${USH_DIR}/prune_stat_files.py file 
+# at line 30
+export USH_DIR="/lfs/h2/emc/vpppg/save/perry.shafran/EVS3/ush/aqm"
```

Comment:
This shouldn't be delivered w/ "perry.shafran". Can this be exported upstream in ecf script?